share/completions/eselect.fish

```diff
@@ -33,12 +34,19 @@ function __fish_complete_eselect_action_options
     __fish_eselect_cmd $cmdl[2..-1] | sed -n -re $findregexp | __fish_sgrep '^    --' | sed -re $parseregexp
 end
 
+function __fish_complete_eselect_php_actions
+    set -l sedregexp 's/^  .([0-9]+).   ([A-Za-z0-9\.]+).?+/\1\t\2/'
+
+    if test (__fish_print_cmd_args_without_options)[3] = 'set'
+        eselect php list (__fish_print_cmd_args_without_options)[-1] | sed -r $sedregexp
```

Comment:
I'm on Archlinux at the moment, that's a bit of the thing. I tend to not want to do make install to get files spread around my filesystem, so I try to run it directly after make which sometimes break ;D

It's probably not fish's fault.

Installing Gentoo is easy, I say. After running it for 8 years. But you probably won't need a full install anyways. Just download stage3, untar, mount (proc, dev, sys), chroot

```
emerge-webrsync
emerge --sync
echo '=app-eselect/eselect-php-0.9.1' > /etc/portage/package.unmask
echo '=app-eselect/eselect-php-0.9.1' > /etc/portage/package.accept_keywords
emerge -aqv php
```

Then you should be ready to fly.